pkgs/top-level/all-packages.nix

```diff
@@ -12222,6 +12222,7 @@ in
   jemalloc = callPackage ../development/libraries/jemalloc { };
 
   jemalloc450 = callPackage ../development/libraries/jemalloc/jemalloc450.nix { };
```

Comment:
Can you also remove this one: 

```suggestion
```

We are not using it anywhere except in mariadb.